tools/translation/health.py

```
@@ -0,0 +1,291 @@
+    return {
+        "body": translatable_body_hash(source_page.frontmatter, source_page.body),
+        "legacy_body": legacy_source_hash(source_page.frontmatter, source_page.body),
+        "localized_metadata": source_localized_metadata_hash(source_page.frontmatter),
+        "structural_metadata": source_structural_metadata_hash(source_page.frontmatter),
+    }
+
+
+def common_page_issues(
+    page: VaultPage,
+    pages: list[VaultPage],
+    language: str,
```

Ruff (0.15.15)

[warning] Unused function argument: `language`

(ARG001)